src/types/fetch.ts

```diff
@@ -3,7 +3,7 @@ import type { FetchRequest, FetchOptions, FetchResponse } from 'ohmyfetch'
 // An interface to extend in a local project
 export interface InternalApi { }
 
-export type NitroFetchRequest = keyof InternalApi | Exclude<FetchRequest, string> | string & {}
+export type NitroFetchRequest = Exclude<keyof InternalApi | Exclude<FetchRequest, string> | string & {}, `/_${string}`|`/api/_${string}`>
```

Comment:
Shall we only exclude from `keyof InternalApi`?

```suggestion
export type NitroFetchRequest = Exclude<keyof InternalApi, `/_${string}`|`/api/_${string}`> | Exclude<FetchRequest, string> | string & {}
```

Reply:
Yes, my mistake, that will be better 👍